rig-neo4j/examples/vector_search_movies_consume.rs

```diff
@@ -32,20 +29,27 @@ mod display;
 
 #[tokio::main]
 async fn main() -> Result<(), anyhow::Error> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::DEBUG)
+        .with_target(false)
+        .init();
+
+    const INDEX_NAME: &str = "moviePlotsEmbedding";
+
     // Initialize OpenAI client
     let openai_api_key = env::var("OPENAI_API_KEY").expect("OPENAI_API_KEY not set");
     let openai_client = Client::new(&openai_api_key);
 
-    let neo4j_uri = env::var("NEO4J_URI").expect("NEO4J_URI not set");
-    let neo4j_username = env::var("NEO4J_USERNAME").expect("NEO4J_USERNAME not set");
-    let neo4j_password = env::var("NEO4J_PASSWORD").expect("NEO4J_PASSWORD not set");
+    let neo4j_uri = "neo4j+s://demo.neo4jlabs.com:7687";
+    let neo4j_username = "recommendations";
+    let neo4j_password = "recommendations";
```

Comment:
Was there a specific reason this was hardcoded in the example?

Reply:
To make the example work out of the box, it uses a demo database from Neo4j!